TripPlanner/View Controllers/TripViewController.m

```diff
@@ -0,0 +1,259 @@
+//
+//  TripViewController.m
+//  TripPlanner
+//
+//  Created by Adrienne Li on 7/13/20.
+//  Copyright © 2020 ahli. All rights reserved.
+//
+
+#import "TripViewController.h"
+#import "Trip.h"
+#import <Parse/Parse.h>
+
+@interface TripViewController ()
+@property (weak, nonatomic) IBOutlet UITextField *titleField;
+@property (weak, nonatomic) IBOutlet UIImageView *tripImageView;
+@property (weak, nonatomic) IBOutlet UITextField *locationField;
+@property (weak, nonatomic) IBOutlet UIDatePicker *startDatePicker;
+@property (weak, nonatomic) IBOutlet UIDatePicker *endDatePicker;
+@property (weak, nonatomic) IBOutlet UITextField *guestsField;
+@property (weak, nonatomic) IBOutlet UILabel *guestList;
+@property (weak, nonatomic) IBOutlet UITextView *descriptionTextView;
+@property (weak, nonatomic) IBOutlet UIBarButtonItem *cancelButton;
+@property (weak, nonatomic) IBOutlet UIBarButtonItem *saveButton;
+@property (strong, nonatomic) NSMutableArray *guests;
+@property (strong, nonatomic) NSMutableArray *guestUsernames;
+
+
+@end
+
+@implementation TripViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    self.guests = [[NSMutableArray alloc] init];
+    self.guestUsernames = [[NSMutableArray alloc] init];
+    self.descriptionTextView.layer.borderWidth = 2;
+}
+
+// get guests array
+- (NSMutableArray *)getGuestsArray:(NSString *)guestsString {
+    // separate guests in guestsString
+    NSArray *guestItems = [guestsString componentsSeparatedByString:@","];
```

Comment:
consider making "," character a class constant, especially in case it gets used in multiple places. This way, you can update it more easily if it needs to change in future.